packages/js-drive/lib/abci/handlers/commitHandlerFactory.js

```diff
@@ -91,22 +89,10 @@ function commitHandlerFactory(
         }
       }
 
-      const documentsTransaction = blockExecutionStoreTransactions.getTransaction('documents');
-
-      const fixCumulativeFeesFeatureFlag = await getLatestFeatureFlag(
-        featureFlagTypes.FIX_CUMULATIVE_FEES, blockHeight, documentsTransaction,
-      );
-
       // Store ST fees from the block to distribution pool
-      if (fixCumulativeFeesFeatureFlag && fixCumulativeFeesFeatureFlag.get('enabled')) {
-        creditsDistributionPool.incrementAmount(
-          blockExecutionContext.getCumulativeFees(),
-        );
-      } else {
-        creditsDistributionPool.setAmount(
-          blockExecutionContext.getCumulativeFees(),
-        );
-      }
+      creditsDistributionPool.setAmount(
```

Comment:
We actually need to use `creditsDistributionPool.incrementAmount`. That was the fix :))